app/tests/components_tests/resources/backends.py

```diff
@@ -75,7 +76,12 @@ def duration(self):
 
     @property
     def usd_cents_per_hour(self):
-        return 100
+        return {
+            GPUTypeChoices.NO_GPU: 50,
+            GPUTypeChoices.T4: 100,
+            GPUTypeChoices.A10G: 150,
+            GPUTypeChoices.V100: 400,
+        }.get(self._requires_gpu_type, 100)
```

Comment:
I excluded A100 and K80 here because I also did not get anything back when I tested these with 'grandchallenge.components.backends.amazon_sagemaker_training.AmazonSageMakerTrainingExecutor'.

Reply:
The changes to this file should be reverted, you can change the executor in your test by adding the `settings` fixture and setting `settings.COMPONENTS_DEFAULT_BACKEND = "grandchallenge.components.backends.amazon_sagemaker_training.AmazonSageMakerTrainingExecutor"`.

The missing A100 makes sense as that is only available in an 8 GPU configuration, but the K80 should be possible as this is available as a single GPY instance (`ml.p2.xlarge`).

However, K80s are quite outdated so will not be used by customers and V100s are also overkill, so users should only be able to select None, T4 and A10G in the challenge request form.